beekeeper-metadata-cleanup/src/test/java/com/expediagroup/beekeeper/metadata/cleanup/service/PagingMetadataCleanupServiceTest.java

```diff
@@ -73,6 +75,9 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 public class PagingMetadataCleanupServiceTest {
 
+  public static final List<String> TABLE_PATHS = List.of("s3://bucket/foo", "s3://bucket/bar", "s3://bucket/foobar");
```

Comment:
Just as personal opinion, these variables always help to understand what are you actually testing, I would recommend you to set examples that help other person to understand the test.

For example, in this case the class could have another variable called TABLE_NAMES = table1, table2, table3 and then Table PATHS can use this variable to create the paths

Reply:
s3://bucket/table/partition1 helps more than s3://foo/bar/2

Reply:
most of the current tests are using those names, i changed it in a few places where i was introducing new variables